src/main/java/org/influxdb/dto/Point.java

```diff
@@ -269,6 +266,11 @@ void setFields(final Map<String, Object> fields) {
     this.fields = fields;
   }
 
+  public Map<String, Object> getFields() {
```

Comment:
Technically the object is mutable in the sense that other properties of the object can be changed after being built.  That aside, could you elaborate on when and why should objects be immutable?